Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```

Comment:
Please use golang:1.14.7 as the base image. 1.13 has been found some vulnerability issues.